tock/tock/templates/hours/timecard_form.html

```diff
@@ -161,7 +159,7 @@
   }
 
   function displayAlerts(selectBox) {
-    var $fieldset = $(selectBox).parent().parent(),
+    var $fieldset = $(selectBox).parent().parent().parent(),
```

Comment:
I think you can pass `parent()` a selector to find a specific parent. For instance:

`$(selectBox).parent('.someClass');`

Reply:
Chaining three parent's like this is a little dangerous because now this code relies on the structure of the HTML to function. So if a developer comes along and changes the hierarchy of the HTML here, they would have to reason to know that this piece of code relies on that certain hierarchy.

Reply:
Totally agreed and the fact that this did break because of the chaining and change in HTML structure with other recent UI updates speaks to this.  I'll look into specifying a selector to get the right parent without relying on the structure entirely, thanks!